api/src/api/user.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const router = express.Router();
+const { errorWrap } = require('../middleware');
+const { sendResponse } = require('../utils/response');
+const { models } = require('../models/index.js');
+
+router.post(
```

Comment:
One more small thing: can you add some documentation above the endpoint that describes what the endpoint does and it's parameters?